src/Slim/Middleware/Session.php

```diff
@@ -122,4 +124,12 @@ protected function startSession()
             session_start();
         }
     }
+
+    private function iniSet($settings) {
```

Comment:
Should this be `proctected`, just in case someone wants to override it?

Reply:
Done